src/App.tsx

```diff
@@ -1,48 +1,44 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import React from 'react';
+import {
+  Navigate, Route, Routes,
+} from 'react-router-dom';
+import { Header } from './components/Header/Header';
+import { TabsPage } from './components/TabsPage/TabsPage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+export const App: React.FC = () => {
+  return (
+    <>
+      <Header />
 
-const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
-      <div className="navbar-menu">
-        <div className="navbar-start">
-          <a href="/" className="navbar-item isActive">Home</a>
-          <a href="/tabs" className="navbar-item isActive">Tabs</a>
-        </div>
-      </div>
-    </nav>
+      <div className="section">
+        <Routes>
+          <Route
+            path="/"
+            element={(
+              <h1 className="title">Home page</h1>
+            )}
+          />
 
-    <div className="section">
-      <h1 className="title">Home page</h1>
-      <h1 className="title">Tabs page</h1>
-      <h1 className="title">Page not found</h1>
+          <Route path="/tabs">
+            <Route index element={<TabsPage />} />
+            <Route path="/tabs/:tabId" element={<TabsPage />} />
+          </Route>
 
-      <div className="tabs is-boxed">
-        <ul>
-          <li data-cy="tab" className="is-active">
-            <a href="#/">Tab 1</a>
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 2</a>
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+          <Route
+            path=":invoiceId"
+            element={(
+              <h1 className="title">Page not found</h1>
+            )}
+          />
 
-      <div className="block" data-cy="tab-content">
-        Please select a tab
+          <Route
+            path="/home"
+            element={<Navigate to="/" />}
```

Comment:
```suggestion
            element={<Navigate to="/" replace />}
```
https://reach.tech/router/api/navigate (at the bottom of the article)